### PR TITLE
Add render flip X or Y axis to position component

### PR DESCRIPTION
**Pull request by kurtome**
This is based on https://github.com/flame-engine/flame/issues/64, I'm using a hack to do the same thing based on the code from the issue already :)

### CODE REVIEW

**Comment by luanpotter** [comment]
@kurtome thanks for your contribution! I don't think it's a hack at all, though it does look like it haha

Can you add an example app & docs (couple of lines under the PositionComponent should suffice) for that new feature?

Code looks good to me :)

File: lib/components/component.dart [hunk 1]
```diff
@@ -128,6 +130,11 @@ abstract class PositionComponent extends Component {
     final double dy = -anchor.relativePosition.dy * height;
     canvas.translate(dx, dy);
 
+    // Handle inverted rendering by moving center and flipping.
+    canvas.translate(width / 2, height / 2);
```
Comment (renancaraujo) [inline lib/components/component.dart]: If both `renderFlipX` and `renderFlipY` are false we should avoid executing all of this code.

[the remark this answers is not in the document]
Comment (kurtome) [reply lib/components/component.dart]: Sounds good, 👍

Comment (kurtome) [reply lib/components/component.dart]: Sounds good, 👍

**Comment by kurtome** [comment]
Updated according to comments, added example, doc and only doing the canvas calculations if flipping is necessary.

**Comment by erickzanardo** [comment]
@kurtome I just noticed that you created this PR from the master branch, would you please rebase your PR with the develop branch? You doing that we can merge it!

**Comment by kurtome** [comment]
@erickzanardo oops, I had the wrong base again, but this time i remembered to branch off of `develop`. Base for this PR updated to `develop` and only seeing my commits.

**Comment by erickzanardo** [comment]
Thanks!

**Pull request merged**